client/app/src/pages/admin/questionnaires/fields/fields.component.ts

```diff
@@ -340,4 +340,37 @@ export class FieldsComponent implements OnInit {
   listenToAddFieldFormTemplate() {
     this.showAddQuestionFromTemplate = false;
   }
+
+  importOption(options: any): void {
+    const fileInput = document.createElement('input');
+    fileInput.type = 'file';
+    fileInput.accept = '.json';
+  
+    fileInput.addEventListener('change', (event: any) => {
+      const file = event.target.files[0];
+      const reader = new FileReader();
+      
+      reader.onload = (e: any) => {
+        const importedOptions = JSON.parse(e.target.result);
+        importedOptions.forEach((option: any) => {
+          this.field.options.push({
+            id: "", // Non capisco chi assegna l'id
```

Comment:
TODO: change the comment to be in english or remove it since the codebase should be in english to enable global peer review.